site/guide/test-sandbox.qmd

```diff
@@ -0,0 +1,18 @@
+---
+title: "{{< fa toolbox >}} Test sandbox (BETA)"
+date: last-modified
+---
+
+<!--- TO DO
+- Ordering of notebooks if we want them to appear in a specific sequence
+--->
+Explore our interactive sandbox to see what tests are available in the ValidMind Developer Framework. These tests cover data validation, model validation, and prompt validation.
+
+::: {.column-screen-right}
+
+```{=html}
+<iframe
+  src="http://test-sandbox.demo.validmind.ai/?embed=true&embed_options=light_theme" width="1800" height="3600" frameborder="0" marginheight="0"></iframe>
```

Comment:
A bit late on my feedback since I was trying to get help to setup SSL on our test sandbox domain. Can you update to this: `https://test-sandbox.demo.validmind.ai/`. HTTPS is in place with a signed certificate so we should be good to go 👍

Reply:
Done!, we now use HTTPS. Re-tested and works as expected.